index.html

```diff
@@ -787,7 +787,7 @@ <h3>
           The [=MiniApp manifest's=] <code><dfn>app_id</dfn></code> member is a [=string=] that identifies the MiniApp univocally. This member is mainly used for package management, and it supports the update and release process of MiniApp versioning.
         </p>
         <p>
-          The format of <code>[=app_id=]</code> is RECOMMENDED to be a [=string=] defined by the following rule:
+          The value of <code>[=app_id=]</code> is RECOMMENDED <a data-link-type="dfn" href="https://infra.spec.whatwg.org/#string-is">to be</a> a [=string=] defined by the following rule:
```

Comment:
`RECOMMENDED` is the equivalent of `SHOULD`, not `MUST`, which means that this is not really a requirement--it's more of a suggestion. If you mean to define a strict namespace (and I think you do to ensure interoperability), you should consider using `MUST` or revising this requirement to just say:

> The value of `app_id` is a string whose contents are defined by the following rule:

---

Note that the link to Infra's "string is" rule makes no sense here. The Infra definition is about string equivalence, not content restriction.

Reply:
I've tried to add it somehow but the 'is' equals to 'is equivalent to' so it does not fit.  I've changed it for `a case-sensitive string`. 

Using SHOULD instead of RECOMMENDED.

Reply:
I probably didn't explain clearly what I meant. 

The Infra reference `string-is` has to do with comparing two strings for equality, not defining what a given string field may consist of. One would use `string-is` to write a requirement like:

> The value of value of the key ***is*** the string `foo`

Here you are trying to define the namespace of `app_id`, so you shouldn't use the `string-is` definition.

If `app_id` can be *any* string and there are no actual requirements on content, I think you need to very careful to spell that out. How will you write the conformance tests? How will implementations deal with apps that do not follow the recommendation? For the miniapp space to work reliably, the spec needs to be super clear.